locals.tf

```diff
@@ -37,6 +37,7 @@ locals {
     AIRFLOW__WEBSERVER__RBAC : var.airflow_authentication == "" ? false : true,
     AIRFLOW__WEBSERVER__AUTH_BACKEND : lookup(local.auth_map, var.airflow_authentication, "")
     AIRFLOW__WEBSERVER__BASE_URL : var.use_https ? "https://${local.dns_record}" : "http://localhost:8080" # localhost is default value
+    AIRFLOW__CORE__FERNET_KEY : "arn:aws:ssm:${var.region}:${data.aws_caller_identity.this.account_id}:parameter/${var.resource_suffix}/airflow/fernet_key"
```

Comment:
You've defined this local (which is just the arn) but it doesn't seem used.

Reply:
Since this says `airflow_variables` and has the other AIRFLOW__CORE variables, I figured I would add it here. So this won't pull the key from parameter store?

Reply:
No, you've defined `AIRFLOW__CORE__FERNET_KEY` as the ARN, so unless airflow inherently knows what to do with that, it won't have any effect.

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html Airflow lets you override its environment variables using AIRFLOW__CORE environment variables.

I'm just trying to make the fernet key available everywhere it might be necessary since whenever the docker container restarts airflow is unable to read the variables we've set in the database for use by the DAGs.

Reply:
Sure but you're doing that already with the other env secret and 'value from' 
Setting here in this local isn't setting the *value* of they key, you're storing a string representation of the ARN. Unless AirFlow explicitly knows how to take a parametstore ARN and pull the value out, I doubt this is doing what you think.

Reply:
That makes sense, I'll remove it then.